src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import jdk.nashorn.internal.objects.annotations.Constructor;
+
+@Entity
+public class Book {
+
```

Comment:
Add the ID attribute to the model (this attribute should be generated automatically).
Typically, the automatically generated ID is done with the @GeneratedValue annotation (strategy = GenerationType.AUTO). But in Postgres when GenerationType.AUTO is used, it generates a shared incremental id between the tables in the schema. This happens only for this database engine. To generate separate identifiers for each table in postgres, use GenerationType.SEQUENCE as follows:
```suggestion
@Id
@GeneratedValue (strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")            
@SequenceGenerator(name = "BOOK_SEQ", sequenceName = "BOOK_SEQ")
private long id;
```